app/package.json

```diff
@@ -31,7 +31,6 @@
     "@types/jest": "^26.0.23",
     "@types/react": "^18.0.21",
     "@types/react-native": "^0.70.6",
-    "@types/react-native-auth0": "2.14.1",
```

Comment:
Do we need this package?

Reply:
I think before I may have added it to extract a class, but I removed that code. The compiler doesn't complain right now.

Reply:
I don't think we need it will add back later if needed.

Reply:
Sounds good, removing packages always feels good haha.